confd/templates/wfm/topology.properties.tmpl

```diff
@@ -1,6 +1,6 @@
 environment.naming.prefix = {{ getv "/kilda_environment_naming_prefix" }}
 
-parallelism.new = 2
+parallelism.new = 1
```

Comment:
Too abstract option... Because it is hotfix, we should hardcode 1 in correct place or add separate option for parallelism of reroute throttling bolt. I vote for hardcoded value with corresponding comment in code.

PS I saw that this option is used only for throttling bolt.

Reply:
Agree. Updated PR.